README.md

```diff
@@ -2,22 +2,32 @@
 [![CI](https://github.com/ansible-collections/community.sops/workflows/CI/badge.svg?event=push)](https://github.com/ansible-collections/community.sops/actions) [![Codecov](https://img.shields.io/codecov/c/github/ansible-collections/community.sops)](https://codecov.io/gh/ansible-collections/community.sops)
 
 <!-- Describe the collection and why a user would want to use it. What does the collection do? -->
+The `community.sops` collection allows integrating [`mozilla/sops`](https://github.com/mozilla/sops) in Ansible.
 
+`mozilla/sops` is a tool for encryption and decryption of files using secure keys (GPG, KMS). It can be leveraged in Ansible to provide an easy to use and flexible to manage way to manage ecrypted secrets' files.
+ 
 ## Tested with Ansible
 
 <!-- List the versions of Ansible the collection has been tested with. Must match what is in galaxy.yml. -->
 
+- `devel`
+
 ## External requirements
 
 <!-- List any external resources the collection depends on, for example minimum versions of an OS, libraries, or utilities. Do not list other Ansible collections here. -->
 
-### Supported connections
-<!-- Optional. If your collection supports only specific connection types (such as HTTPAPI, netconf, or others), list them here. -->
+You will need to install [`sops`](https://github.com/mozilla/sops) manually before using plugins provided by this
```

Comment:
Is there a link to that Sops installation page?
Is there a minimum (or maximum) version of Sops that this collection requires?

Reply:
This is a very interesting question. I would love to have a test suite where multiple `sops` versions are tested, but I'm still struggling with the tests :) 

But as the CLI interface is not expected to change, I would expect any `3.x` `sops` release to be working. I'll add it to the README.

Reply:
For the initial 0.1.0 release, it's totally fine if not everything is perfect :)